adapter/src/components/ErrorBoundary.js

```diff
@@ -12,7 +13,7 @@ const replaceNewlinesWithBreaks = text =>
         .split('\n')
         .reduce((out, line, i) => [...out, line, <br key={i} />], [])
 
-export class FatalErrorBoundary extends Component {
+export class ErrorBoundary extends Component {
```

Comment:
Awesome. Should we maybe (outside of this change) move this to @dhis2/ui so that it can be used in apps as well? cc @jenniferarnesen

Reply:
I think so!